docker-java-core/src/main/java/com/github/dockerjava/core/DockerContextMetaFile.java

```diff
@@ -52,6 +43,16 @@ public static Optional<DockerContextMetaFile> resolveContextMetaFile(ObjectMappe
         return Optional.ofNullable(loadContextMetaFile(objectMapper, path));
     }
 
+    public static Optional<File> resolveContextTLSFile(File dockerConfigPath, String context) {
+        final File path = dockerConfigPath.toPath()
+            .resolve("contexts")
+            .resolve("tls")
+            .resolve(metaHashFunction.hashString(context, StandardCharsets.UTF_8).toString())
+            .resolve("docker")
+            .toFile();
+        return  Optional.ofNullable(path).filter(File::exists);
+    }
```

Comment:
What I can see by doing `docker context inspect desktop-linux` is

```
[
    {
        "Name": "desktop-linux",
        "Metadata": {
            "Description": "Docker Desktop"
        },
        "Endpoints": {
            "docker": {
                "Host": "unix:///Users/eddumelendez/.docker/run/docker.sock",
                "SkipTLSVerify": false
            }
        },
        "TLSMaterial": {},
        "Storage": {
            "MetadataPath": "/Users/eddumelendez/.docker/contexts/meta/fe9c6bd7a66301f49ca9b6a70b217107cd1284598bfc254700c989b916da791e",
            "TLSPath": "/Users/eddumelendez/.docker/contexts/tls/fe9c6bd7a66301f49ca9b6a70b217107cd1284598bfc254700c989b916da791e"
        }
    }
]
```

and the tls can be retrieved using the `Storage > TLSPath` in the json file, plus adding the `docker` folder at the end, right? or is there any other motivation behind this change?

Reply:
Try to find it on disk, as I wrote this output is composed by docker cli, on this there is folder, but this information is not in json file - it was my initial idea, which was proven to not work, so there is this PR to fix it

Reply:
got it! I forgot that this is not stored 👍🏽

I’ll take another look later today and merge it.

Thanks!